src/AzureAuth.Test/AzureAuth.Test.csproj

```diff
@@ -8,8 +8,7 @@
     <TargetFramework>net6.0</TargetFramework>
     <IsPackable>false</IsPackable>
 
-    <!-- Stylecop warnings as errors flag for build to fail -->
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>False</TreatWarningsAsErrors>

```

Comment:
Do we want to downgrade to warnings in non-test projects too?

Reply:
Yes. I would like to delete StyleCop unless we can treat everything as a warning instead of a hard error.

Reply:
I agree. And I'd like to do a sweep all at once, I'll revert here.